library/src/main/scala/scala/scalajs/js/typedarray/TypedArray.scala

```diff
@@ -65,6 +65,17 @@ trait TypedArray[T, Repr] extends ArrayBufferView with js.Iterable[T] {
  *  Static information that exists for any concrete TypedArray
  */
 @js.native
-trait TypedArrayStatic extends js.Object {
+trait TypedArrayStatic[T, Repr] extends js.Object {
   val BYTES_PER_ELEMENT: Int = js.native
+
+  /** Returns a new array from a set of elements. */
+  def of(items: T*): Repr = js.native
+
+  /** Creates an array from an `iterable` object. */
+  def from[E](iterable: js.Iterable[E],
+      mapFn: js.Function1[E, T] = ???): Repr = js.native
```

Comment:
This definition is unsound, because I can call it with:
```scala
val a = js.Array[String]("foo", "bar")
val b = js.typedarray.Int32Array.from(a)
```
and the compiler won't complain (it will simply instantiate `E = String`).

You should keep a separate overload with only 1 argument and no type parameter, for the case where no map function is provided.

Reply:
Ah, I did not know it is unsound.

Fixed in https://github.com/scala-js/scala-js/commit/a7dc30d9c1e6a518e8c12615b41c51d680aa5f0a
(commits are squashed as requested always)